### PR TITLE
Move resolve all to isolated action

### LINKED ISSUE #2312: Resolve All and Filtered Search can Lead to UI Failure
From a support ticket:

> I just accidentally resolved all the issues in my entire app due to what seems like poor UX.
>
> I searched for 2 specific issues I wanted to resolve, checked their boxes and then clicked the check mark to resolve.
>
> When it asked me if I wanted to resolve all incidents, I assumed it meant all 1000+ instances of the 2 incidents I had selected. I would not expect it to operate on any incidents that were not visible in the UI.

This happens because I assume the dialog triggers when you select all visible issues, but the resolve all does not take into account the current query.  That dialog should probably not come up unless the filter is empty or the resolve should take the filter into account.

### PR DESCRIPTION
**Pull request by dcramer**
Fixes GH-2312

### CODE REVIEW

File: src/sentry/static/sentry/app/views/stream/actionLink.jsx [hunk 1]
```diff
@@ -116,7 +116,10 @@ const ActionLink = React.createClass({
 
         <Modal show={this.state.isModalOpen} title="Please confirm" animation={false} onHide={this.handleToggle}>
           <div className="modal-body">
-            <p><strong>Are you sure that you want to {this.props.actionLabel}?</strong></p>
+            <p><strong>Are you sure that you want to {actionLabel}?</strong></p>
```
Comment (dcramer) [inline src/sentry/static/sentry/app/views/stream/actionLink.jsx]: this was broken at some point, so its fixed in this change as well

**Comment by dcramer** [comment]
the actionAll bits are no longer used, so we could remove those, but wanted to keep this patch small

**Comment by dcramer** [comment]
@benvinegar lol if you think i have any clue how to write reliable tests

**Comment by benvinegar** [comment]
I wasn't expecting you to do so as part of this PR. I also know we don't have a good base for a lot of components. But, I feel I should make half-hearted comment :)

**Pull request merged**